python/fapolicy_analyzer/util/xdg_utils.py

```diff
@@ -0,0 +1,67 @@
+import os
+
+
+def xdg_state_dir_prefix(strBaseName):
```

Comment:
This and `xdg_data_dir_prefix` methods seem to repeat a lot of code.  Maybe create a private method to handle it that takes either `state` or `share` as a 2nd argument and call it from these 2 methods.

Reply:
Will create a utility helper function...

Reply:
Done